src/Button/types.ts

```diff
@@ -48,7 +54,15 @@ export type ButtonProps = {
    * The trailing icon comes after button content
    */
   trailingIcon?: React.FunctionComponent<React.PropsWithChildren<IconProps>>
+  /**
+   * Trailing action appears to the right of the trailing visual and is always locked to the end
+   */
+  trailingAction?: React.FunctionComponent<React.PropsWithChildren<IconProps>>
```

Comment:
This name is a little confusing. It makes it sound like it's a button within the button. The best alternative I could come up is `endIcon`.

Reply:
This one was [discussed](https://github.com/github/primer/discussions/1016) awhile ago here, and is also implemented already in PVC. My PR for the next major release will change `trailingIcon` to `trailingVisual` as well, which I think will better align with `trailingAction`. It's also part of the ActionList API which hasn't fully made its way into PRC.